files.go

```diff
@@ -351,6 +351,50 @@ func (c *Files) Download(in *DownloadInput) (out *DownloadOutput, err error) {
 	return
 }
 
+const (
+	// GetThumbnailFormatJPEG specifies a JPG thumbnail
+	GetThumbnailFormatJPEG = "jpeg"
+	// GetThumbnailFormatPNG specifies a PNG thumbnail
+	GetThumbnailFormatPNG = "png"
+)
+
+const (
+	// GetThumbnailSizeW32H32 specifies a size of 32 by 32 px
+	GetThumbnailSizeW32H32 = "w32h32"
+	// GetThumbnailSizeW64H64 specifies a size of 64 by 64 px
+	GetThumbnailSizeW64H64 = "w64h64"
+	// GetThumbnailSizeW128H128 specifies a size of 128 by 128 px
+	GetThumbnailSizeW128H128 = "w128h128"
+	// GetThumbnailSizeW640H480 specifies a size of 640 by 480 px
+	GetThumbnailSizeW640H480 = "w640h480"
+	// GetThumbnailSizeW1024H768 specifies a size of 1024 by 768 px
+	GetThumbnailSizeW1024H768 = "w1024h768"
+)
+
+// GetThumbnailInput request input.
+type GetThumbnailInput struct {
+	Path   string `json:"path"`
+	Format string `json:"format"`
```

Comment:
might be worth making these types like the others (https://github.com/tj/go-dropbox/blob/master/files.go#L23-L30), could be annoying if one isn't defined but helps with typos at least I suppose!